mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.erb

```diff
@@ -1419,6 +1428,18 @@ func resourceTable(d *schema.ResourceData, meta interface{}) (*bigquery.Table, e
 		table.TableConstraints = tableConstraints
 	}
 
+  <% unless version == 'ga' -%>
+	if v, ok := d.GetOk("resource_tags"); ok {
```

Comment:
non-blocking, but we could use `ExpandStringMap` here: https://github.com/GoogleCloudPlatform/magic-modules/blob/60a85738f569bbfa2731097270db8b19d5dcfebd/mmv1/third_party/terraform/tpgresource/utils.go#L231

(same result though 🤷 )

Reply:
Done.